lib/cli/input/prefs.js

```diff
@@ -87,16 +87,20 @@ const changeGitlab = (host) => {
 };
 
 const changeServer = async (options) => {
-  let { host, port, type } = options;
+  const { host, port } = options;
+  let { type } = options;
   if (!type) {
     const typePrompt = prefPrompts.getServerTypePrompt();
-    type = (await inquirer.prompt(typePrompt)).type;
+    ({ type } = (await inquirer.prompt(typePrompt)));
```

Comment:
why brackets?

Reply:
We have discussed this, sir, but for having everything in a single thread; the MDN documentation says: 

> 

    The parentheses ( ... ) around the assignment statement are required when using object literal destructuring assignment without a declaration.

    {a, b} = {a: 1, b: 2} is not valid stand-alone syntax, as the {a, b} on the left-hand side is considered a block and not an object literal.

    However, ({a, b} = {a: 1, b: 2}) is valid, as is var {a, b} = {a: 1, b: 2}

    NOTE: Your ( ... ) expression needs to be preceded by a semicolon or it may be used to execute a function on the previous line.